projects/plugins/social/src/js/components/support-section/index.js

```diff
@@ -1,13 +1,20 @@
-import { Text, Container, getRedirectUrl } from '@automattic/jetpack-components';
+import {
+	Text,
+	Container,
+	getRedirectUrl,
+	useBreakpointMatch,
+} from '@automattic/jetpack-components';
 import { ExternalLink } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 import { Icon, lifesaver } from '@wordpress/icons';
+import classNames from 'classnames';
 import { STORE_ID } from '../../store';
 import IconText from '../icon-text';
 import styles from './styles.module.scss';
 
 const SupportSection = () => {
+	const [ isAtLeastMedium ] = useBreakpointMatch( 'md', '>=' );
```

Comment:
Is there a difference between `useBreakpointMatch( 'md', '>=' )` and `useBreakpointMatch( 'md' );`?

Reply:
Yeah, first I used `useBreakpointMatch( 'md' );` but that's only true if the viewport is exactly medium! To match the `@media` `min-width` queries we need the _at least_ medium ones.

Reply:
Ah, so it probably works for `isLg` because there's nothing bigger than `lg`, makes sense!